SQL/Archive/2018-05-28_publication_schema.sql

```diff
@@ -0,0 +1,160 @@
+SET FOREIGN_KEY_CHECKS=0;
+-- Publication Status
+DROP TABLE IF EXISTS publication_status;
+CREATE TABLE `publication_status` (
+  `PublicationStatusID` int(2) unsigned NOT NULL,
+  `Label` varchar(255) NOT NULL,
+  PRIMARY KEY(`PublicationStatusID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+DELETE FROM publication_status;
+INSERT INTO publication_status VALUES (1, 'Pending');
+INSERT INTO publication_status VALUES (2, 'Approved');
+INSERT INTO publication_status VALUES (3, 'Rejected');
+
+-- Main table
+DROP TABLE IF EXISTS publication;
+CREATE TABLE `publication` (
+    `PublicationID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    `PublicationStatusID` int(2) unsigned NOT NULL default 1,
+    `UserID` int(10) unsigned NOT NULL,
+    `RatedBy` int(10) unsigned,
+    `DateProposed` date NOT NULL,
+    `DateRated` date default NULL,
+    `Title` varchar(255) NOT NULL,
+    `LeadInvestigator` varchar(255) NOT NULL,
+    `LeadInvestigatorEmail` varchar(255) NOT NULL,
+    `RejectedReason` varchar(255) default NULL,
+    `Description` text NOT NULL,
+    PRIMARY KEY(`PublicationID`),
+    CONSTRAINT `FK_publication_1` FOREIGN KEY(`UserID`) REFERENCES `users` (`ID`),
+    CONSTRAINT `FK_publication_2` FOREIGN KEY(`RatedBy`) REFERENCES `users` (`ID`),
+    CONSTRAINT `FK_publication_3` FOREIGN KEY(`PublicationStatusID`) REFERENCES `publication_status` (`PublicationStatusID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+-- Separate table for Keywords
+DROP TABLE IF EXISTS publication_keyword;
+CREATE TABLE `publication_keyword` (
+  `PublicationKeywordID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Label` varchar(255) NOT NULL,
+  PRIMARY KEY(`PublicationKeywordID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+
+-- Publication - Keyword relational table
+DROP TABLE IF EXISTS publication_keyword_rel;
+CREATE TABLE `publication_keyword_rel` (
+  `PublicationID` int(10) unsigned NOT NULL,
+  `PublicationKeywordID` int(10) unsigned NOT NULL,
+  CONSTRAINT `UK_publication_keyword_1` UNIQUE (PublicationID, PublicationKeywordID),
```

Comment:
This could be made a primary key instead of "only" unique. I think it is technically the same but stating that the two columns are the primary key is more explicit